### PR TITLE
Add Lovoo

### PR DESCRIPTION
**Pull request by Maiskolbenschaeler**
Adds the following events or companies:
 - Lovoo

### Checklist

 - [x] I have updated the event or company counts
 - [x] I have put the most recent relevant date in the "Last Update" column
 - [ ] I have linked to the article about the change, not the company's website

### CODE REVIEW

File: _data/companies.yml [hunk 1]
```diff
@@ -1222,3 +1222,10 @@
   visitors: "?"
   events: Restricted
   last_update: 2020-03-11
+  
+- name: Lovoo 
+  wfh: Required 
+  travel: Restricted 
+  visitors: Restricted 
+  events: Restricted 
+  last_update: 2020-03-13
```
Comment (drewbrew) [inline _data/companies.yml]: Is that a typo or are you based on a Pacific island?

Comment (Maiskolbenschaeler) [reply _data/companies.yml]: typo, should be 2020-03-12

Comment (drewbrew) [reply _data/companies.yml]: No problem. I fixed it while taking care of the merge conflict.

Comment (Maiskolbenschaeler) [reply _data/companies.yml]: thx :)

**Pull request merged**